src/common/models/cluster/cluster.ts

```diff
@@ -84,8 +70,11 @@ function ensureNotTiny(v: number): void {
   }
 }
 
+function oldHostParameter(cluster: any): string {
+  return cluster.host || cluster.druidHost || cluster.brokerHost;
```

Comment:
Good catch, I didn't know that druidHost and brokerHost is implicitly supported.